tests/menubar-editor/data/test.template

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>%TITLE%</title>
+<link rel="author" title="Valerie Young" href="mailto:valerie@bocoup.com">
+%REFERENCES%
+<script type="module">
+  import { verifyATBehavior, displayTestPageAndInstructions } from "../resources/aria-at-harness.mjs";
+
+  verifyATBehavior({
+    %SETUP_TEST_PAGE%
+    applies_to: ["%APPLIES_TO%"],
```

Comment:
In a different PR we are about to merge (https://github.com/w3c/aria-at/pull/61), we are change this list to a single string, the mode will be "reading" or "interaction". I'll merge that PR this afternoon, can we remove the brackets here?